cirq_superstaq/daily_integration_test.py

```diff
@@ -129,7 +141,9 @@ def test_cq_compile(service: cirq_superstaq.Service) -> None:
     )
 
     out = service.cq_compile(circuit)
-    assert out.circuit == compiled_circuit
+    cirq.testing.assert_circuits_with_terminal_measurements_are_equivalent(
+        out.circuit, compiled_circuit, atol=1e-08
```

Comment:
this one should work too
```suggestion
        out.circuit, circuit, atol=1e-08
```

Reply:
Done.